scene/2d/animated_sprite.cpp

```diff
@@ -490,6 +490,34 @@ Point2 AnimatedSprite::get_offset() const {
 	return offset;
 }
 
+void AnimatedSprite::set_anchor(const Point2 &p_anchor) {
+
+	anchor = p_anchor;
+	anchor.x = CLAMP(anchor.x, -1, 1);
+	anchor.y = CLAMP(anchor.y, -1, 1);
+
+	Point2 calc_anchor = anchor;
+
+	if (centered) {
+		calc_anchor.x = CLAMP(anchor.x, -0.5, 0.5);
+		calc_anchor.y = CLAMP(anchor.y, -0.5, 0.5);
```

Comment:
No, I wanted to clamp the values in range (-0.5 to 0.5) so if a sprite is centered so it would stay at the edges and not go further.

If I remove clamping, should I do some calculations here so that the position stays the same if the sprite is centered or not? For example, a anchor of (0, 1) would position the node at the sprite's feet when not centered. If centered though, you would have to use (0, 0.5) to get that same position. Should I detect if a sprite is centered and make adjustments as necessary to keep the positioning the same?